tests/spread/integration/golang-1.22-go/test_build-tools.sh

```diff
@@ -0,0 +1,6 @@
+chroot "${rootfs}" /usr/lib/go-1.22/bin/go tool asm -V
+(chroot "${rootfs}" /usr/lib/go-1.22/bin/go tool buildid 2>&1 || true) | grep "usage"
```

Comment:
nit: you need this kind of `(... || true)` pattern only when there is `grep -q` or `head` or some other util which exists early with SIGPIPE. it's either not-needed here, or make it `grep -q`